packages/contracts/contracts/core/erc165/AdaptiveERC165.sol

```diff
@@ -19,9 +19,15 @@ contract AdaptiveERC165 is ERC165 {
     error AdapERC165UnkownCallback(bytes32 magicNumber);
 
     // Events
-    event RegisteredStandard(bytes4 interfaceId);
-    event RegisteredCallback(bytes4 sig, bytes4 magicNumber);
-    event ReceivedCallback(bytes4 indexed sig, bytes data);
+
+    /// @dev Emmitted when a new standard is registred and assigned to `interfaceId`
+    event StandardRegistered(bytes4 interfaceId);
+
+    /// @dev Emmitted when a callback is registered
+    event CallbackRegistered(bytes4 sig, bytes4 magicNumber);
+
+    /// @dev Emmitted when a callback is received
+    event CallbackReceived(bytes4 indexed sig, bytes data);
```

Comment:
The `/// @dev` comments don't add much information to it.
Add natspec for the parameters, or create a separate issue for this.

Reply:
Created new subtask to tackle this.